frontend/pages/Homepage/HostsSummary/HostsSummary.tsx

```diff
@@ -17,12 +16,22 @@ interface IRootState {
       data: {
         [id: number]: {
           count: number;
+          name: string;
+          label_type: string;
         };
```

Comment:
```suggestion
        [id: number]: ILabel,
```

Then we can also update that interface to include the values we use here.